contrib/epee/include/net/local_ip.h

```diff
@@ -27,13 +27,22 @@
 
 #pragma once
 
+#include "int-util.h"
+
+// IP addresses are kept in network byte order
+// Masks below are little endian
+// -> convert from network byte order to host byte order before comparing
+
 namespace epee
 {
   namespace net_utils
   {
     inline
     bool is_ip_local(uint32_t ip)
     {
+#if BYTE_ORDER == BIG_ENDIAN
```

Comment:
Isn't this `#if` redundant? The compiler should catch the nop of self-assigning an integer.

Reply:
It does appears to be, yes, I'll remove.